IdentityCore/tests/automation/MSIDTestAccountsProvider.h

```diff
@@ -32,6 +32,7 @@
 - (instancetype)initWithClientCertificateContents:(NSString *)certificate
                               certificatePassword:(NSString *)password
                          additionalConfigurations:(NSDictionary *)additionalConfigurations
+                                  appInstallLinks:(NSDictionary *)appInstallLinks
```

Comment:
nit: can you put a comment - sample payload for this?

Reply:
Done 👍